knowledge-base/actions/dorny/test-reporter/action-security.yml

```diff
@@ -0,0 +1,14 @@
+name: Test Reporter
+github-token:
+  action-input:
+    input: token
+    is-default: true
+  permissions:
```

Comment:
update reasons to:
- checks-reason: to create check run
- actions-reason: to download artifact
- contents-reason: to get commit for testing